extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -1491,11 +1491,20 @@ static public function onUnwatchArticle( &$user, &$article ) {
 		// Since we need a real Title backed by a DB row, we need to reconstruct a title object
 		// if the current one looks fake.  If this already is a real Title the normal unwatch handling
 		// that called this hook will take care of this for us.
-		if ( ( $article->getId()  == 0 ) && preg_match( '/^(\d+)$/', $title->getText(), $matches ) ) {
+		if ( ( $article->getId() == 0 ) && preg_match( '/^(\d+)$/', $title->getText(), $matches ) ) {
 			$id = $matches[1];
-			$title = Title::newFromID( $id );
+			$realTitle = Title::newFromID( $id );
```

Comment:
Keep `$title`, which is the fake `Title` the wall code constructs, separate from the real `Title` object created on this line.